src/component/base/DraftEditor.react.js

```diff
@@ -368,13 +368,19 @@ class DraftEditor extends React.Component<DraftEditorProps, State> {
     const alreadyHasFocus = editorState.getSelection().getHasFocus();
     const editorNode = ReactDOM.findDOMNode(this.refs.editor);
 
-    if (!(editorNode instanceof HTMLElement)) {
+    if (!editorNode) {
+      // once in a while people call 'focus' in a setTimeout, and the node has
+      // been deleted, so it can be null in that case.
       return;
     }
 
     const scrollParent = Style.getScrollParent(editorNode);
```

Comment:
correct me if I'm wrong, but Style.getScrollParent has 1 param & that is `node`, so we don't want to call it unless we have an `HTMLElement`, right? It doesn't matter either way (code below), but if we were writing a unit test for this, then it'd be REALLY hard to reach that invariant...

```
getScrollParent: function getScrollParent(node) {
	    if (!node) {
	      return null;
	    }
            ...
```